src/node.cc

```diff
@@ -3111,6 +3111,7 @@ static void PrintHelp() {
          "  --track-heap-objects  track heap object allocations for heap "
          "snapshots\n"
          "  --v8-options          print v8 command line options\n"
+         "  --cipher-list=list    use an alternative default cipher list\n"
```

Comment:
This help output will also show regardless of whether or not node was compiled with SSL, is that on purpose?